cmake/modules/BuildFIO.cmake

```diff
@@ -38,5 +38,5 @@ function(build_fio)
   add_dependencies(fio fio_ext)
   set_target_properties(fio PROPERTIES
     INTERFACE_INCLUDE_DIRECTORIES ${source_dir}
-    INTERFACE_COMPILE_OPTIONS "-include;${source_dir}/config-host.h;$<$<COMPILE_LANGUAGE:C>:-std=gnu99>$<$<COMPILE_LANGUAGE:CXX>:-std=gnu++17>")
+    INTERFACE_COMPILE_OPTIONS "-include;${source_dir}/config-host.h;$<$<COMPILE_LANGUAGE:C>:-std=gnu99>$<$<COMPILE_LANGUAGE:CXX>:>")
```

Comment:
the commit where i introduced these interface options was c190fee53d23212faf22b26df33346b393e11b8d. i wanted to populate `CXX_EXTENSIONS=ON` to the targets linking against `fio`. could you please preserve this ? probably we can use 

```cmake
set_target_properties(fio PROPERTIES
  CXX_EXTENSIONS ON)
```
?
`$<$<COMPILE_LANGUAGE:CXX>:>` is a generator expression, which expanded to an empty string. if we want top drop `-std=gnu++17`, we might want to drop the whole expression.

Reply:
ok, will update;  thanks Kefu!